docs/core/porting/index.md

```diff
@@ -13,35 +13,96 @@ You might have code that currently runs on the .NET Framework that you're intere
 
 ## Overview of the porting process
 
+Porting to .NET Core (or .NET Standard) from .NET Framework for many projects is relatively straight forward. There are a number of changes that are required, but many of them follow the patterns outlined below. Projects where the app-model is available in .NET Core (such as libraries, console apps, and desktop applications) usually require little changes. Projects that require a new app model, such as moving to ASP.NET Core from ASP.NET, require a bit more work, but many patterns have analogs that can be used during the conversion. This document should help with identifying the main strategies that have been employed by users to successfully convert their code bases to target .NET Standard or .NET Core and will address the conversion at two levels: solution-wide and project specific. See the links at the bottom for directions on app-model specific conversions.
+
+We recommend you use the following process when porting your project to .NET Core. Each of these steps introduces potential places for behavior changes, so ensure that you adequately test your library or application before continuing on to later steps. The first steps are to get your project ready for a switch to .NET Standard or .NET Core. If you have unit tests, it's best to convert them first so that you can continue testing changes in the product you're working on. Because porting to .NET Core is such a significant change to your codebase, it's highly recommended to port your test projects so that you can run tests as you port your code over. MSTest, xUnit, and NUnit all work on .NET Core.
+
+## Getting started
+
+The following tools will be used throughout the process:
+
+- Visual Studio 2019
+- Download [.NET Portability Analyzer](../../standard/analyzers/portability-analyzer.md)
+- _Optional_ Install [dotnet try-convert](https://github.com/dotnet/try-convert)
+
+## Porting a solution
+
+When there are multiple projects in a solution the porting can seem more complicated because you must address projects in a specific order. The conversion process should be a bottom-up approach, where the projects with no dependencies on other projects in the solution are converted first, and continue up through the whole solution.
+
+In order to identify the order projects should be migrated, you can use the following tools:
+
+- [Dependency Diagrams in Visual Studio](/visualstudio/modeling/create-layer-diagrams-from-your-code) can create a directed graph of the code in a solution.
```

Comment:
Does the feature work for C++/CLI and F# projects? Which VS component or workload is also required to install these? I would suspect the majority of customers don't have the tooling installed unless it's a default in the most popular workloads.